keyboards/sixkeyboard/rules.mk

```
@@ -29,3 +29,4 @@ BLUETOOTH_ENABLE = no	# Enable Bluetooth with the Adafruit EZ-Key HID
 CUSTOM_MATRIX = yes
 
 SRC = matrix.c
```

```suggestion
SRC += matrix.c
```
Entirely unrelated, but may as well fix this :P